pkgs/os-specific/linux/systemd/default.nix

```
@@ -49,9 +51,13 @@ assert withImportd ->
   ( curl.dev != null && zlib != null && xz != null && libgcrypt != null
   && gnutar != null && gnupg != null);
 
+# Homed needs cryptsetup
+assert withHomed -> ( withCryptsetup != false );
```

```suggestion
assert withHomed -> withCryptsetup;
```
`x : x != false` is the identity function for booleans :).